libr/core/project.c

```diff
@@ -354,6 +354,15 @@ R_API RThread *r_core_project_load_bg(RCore *core, const char *prjName, const ch
 	ps->rcPath = strdup (rcPath);
 	RThread *th = r_th_new (projectLoadBackground, ps, false);
 	r_th_start (th, true);
+#if 0
```

Comment:
can we remove this?

Reply:
sure ... in fact was not sure it would go without.